# This service computes the effector to camera transformation from 
# data provided on camera_object and world_effector topics
#

---
geometry_msgs/Transform effector_camera